packages/packages/user-preferences/src/hooks/use-action-props.ts

```diff
@@ -0,0 +1,12 @@
+import { __ } from '@wordpress/i18n';
+import { openRoute } from '@elementor/v1-adapters';
+import ToggleRightIcon from '../icons/toggle-right-icon';
+
+// TODO: Use return type from the TopBar package once merged.
+export default function useActionProps() {
+	return {
+		icon: ToggleRightIcon,
+		title: __( 'User Preferences', 'elementor' ),
+		onClick: () => openRoute( 'panel/editor-preferences' ),
```

Comment:
Should it be toggle? with state and closed when click again?

Reply:
Done